README.md

```diff
@@ -53,6 +49,31 @@ The hashes, which are the first two sets of two characters of the MD5 hash of th
 are required in order to prevent a single directory from containing too many subdirectories
 (to avoid problems with some filesystems).
 
+## Usage of the Docker Image
+
+Build the docker image:
+
+    docker build -t flickr-cli .
+
+Run the ```application.php``` using the docker image built before:
+
+		docker run -it --rm -u $(id -u):$(id -g) -v $PWD:/mnt flickr-cli
```

Comment:
I think a common usage of the docker setup would be to set up a quick command to upload a directory or backup one's Flickr photos, so perhaps we should give these commands more from that point of view? e.g. as they'd need to be entered as a command alias or similar. Or maybe we provide a launch script?